CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 None
+<a name="4.1.3"</a>
+## [3.0.2] 2021-11-3
```

Comment:
This is not the correct value for the version - should be `4.1.3`
In addition we need to update the actual version number in the `build.gradle` file

Reply:
You're referring to the version in square brackets correct?

Reply:
Correct

Reply:
Ok cool went ahead and updated that version along with the version in the `build.grandle` file